L1Trigger/L1THGCal/src/backend/HGCalMulticlusteringHistoImpl.cc

```diff
@@ -215,16 +220,84 @@ std::vector<GlobalPoint> HGCalMulticlusteringHistoImpl::computeMaxSeeds( const H
                 isMax &= MIPT_seed>MIPT_SW;
                 isMax &= MIPT_seed>MIPT_NW;
 
+                if(isMax){
+                    float ROverZ_seed = kROverZMin_ + (bin_R+0.5) * (kROverZMax_-kROverZMin_)/nBinsRHisto_;
+                    float phi_seed = -M_PI + (bin_phi+0.5) * 2*M_PI/nBinsPhiHisto_;
+                    float x_seed = ROverZ_seed*cos(phi_seed);
+                    float y_seed = ROverZ_seed*sin(phi_seed);
+
+                    seedPositions.emplace_back( std::make_pair( GlobalPoint(x_seed,y_seed,z_side), MIPT_seed) );
```

Comment:
`make_pair` is not needed
And `seedPositions` is not a good name anymore since it stores also the seed value.